assembl/static2/js/app/components/cookies/cookieSwitch.jsx

```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as React from 'react';
+import { Translate, I18n } from 'react-redux-i18n';
+
+import SwitchButton from '../common/switchButton';
+
+type cookieToggleProps = {
+  cookieIsMatomo: boolean,
+  matomoOptOutLink: ?string,
+  toggleSwitch: Function,
+  accepted: boolean,
+  name: string,
+  required: boolean
+}
+
+const cookieSwitch = ({ cookieIsMatomo, matomoOptOutLink, toggleSwitch, accepted, name, required }: cookieToggleProps) => {
```

Comment:
please use an uppercase for components first letter

Reply:
I thought you don't do that for functional components and only use an uppercase when you render it (<CookieSwitch />) but there seems to be a debate :) @pascalhuynh

Reply:
Like I said, I don't agree with this style change, https://github.com/assembl/assembl/pull/503#commitcomment-30357255

Reply:
Allright 😃

Reply:
@vincentfretin oops, I didn't see your tag on The previous PR. I am 100% on the side of Capitalizing functional components. We do it for class based components, why not for functional? And agreed, I search for components with case sensitivity, it makes finding things quicker. I'll copy/paste this on the other thread.